pyccl/halos/profiles.py

```diff
@@ -267,17 +275,167 @@ def cumul2d(self, cosmo, r_t, M, a, mass_def=None):
             are scalars, the corresponding dimension will be
             squeezed out on output.
         """
-        if getattr(self, '_cumul2d', None):
+        if getattr(self, "_cumul2d", None):
             s_r_t = self._cumul2d(cosmo, r_t, M, a, mass_def)
         else:
-            s_r_t = self._projected_fftlog_wrap(cosmo, r_t, M,
-                                                a, mass_def,
-                                                is_cumul2d=True)
+            s_r_t = self._projected_fftlog_wrap(
+                cosmo, r_t, M, a, mass_def, is_cumul2d=True
+            )
         return s_r_t
 
-    def _fftlog_wrap(self, cosmo, k, M, a, mass_def,
-                     fourier_out=False,
-                     large_padding=True):
+    def sigma_critical(self, cosmo, a_lens, a_src):
+        """Returns the critical surface mass density.
+
+        .. math::
+         \\Sigma_{\\mathrm{crit}} = \\frac{c^2}{4\\pi G}
+          \\frac{D_{\\rm{s}}}{D_{\\rm{l}}D_{\\rm{ls}}},
+           where :math:`c` is the speed of light, :math:`G` is the
+           gravitational constant, and :math:`D_i` is the angular diameter
+           distance. The labels :math:`i =` s, l and ls denotes the distances
+           to the source, lens, and between source and lens, respectively.
+            Args:
```

Comment:
There should be a space between the text and `Args`, and `Args` should be unindented once, otherwise it won't get recognized by coding software.